spec/intl.html

```diff
@@ -190,7 +193,9 @@ <h1>InitializeDateTimeFormat ( _dateTimeFormat_, _locales_, _options_ )</h1>
         1. <ins>If _needDefaults_ is *true*, then</ins>
           1. <ins>For each element _field_ of « *"year"*, *"month"*, *"day"*, *"hour"*, *"minute"*, *"second"* » in List order, do</ins>
             1. <ins>Set _expandedOptions_.[[&lt;_field_&gt;]] to *"numeric"*.</ins>
-        1. Set _dateTimeFormat_.[[Pattern]] to <del>_pattern_</del><ins>GetDateTimeFormatPattern(_matcher_, _expandedOptions_, _formats_, _hc_)</ins>.
+        1. <ins>Let _bestFormat_ be GetDateTimeFormatPattern(_matcher_, _limitedOptions_, _formats_, _hc_)</ins>.
```

Comment:
`_limitedOptions_` used before define, so it's hard to follow this bit.

Reply:
this is a bug, I accidentally used `_limitedOptions_` instead of `_expandedOptions_`. Thanks, will fix.